pkg/ccl/multiregionccl/datadriven_test.go

```diff
@@ -114,9 +114,13 @@ func TestMultiRegionDataDriven(t *testing.T) {
 	defer log.Scope(t).Close(t)
 
 	skip.UnderRace(t, "flaky test")
-
 	ctx := context.Background()
 	datadriven.Walk(t, datapathutils.TestDataPath(t), func(t *testing.T, path string) {
+
+		if strings.Contains(path, "secondary_region") {
+			skip.UnderStress(t, "flaky test")
```

Comment:
can you add a link to https://github.com/cockroachdb/cockroach/issues/92235 here, so people know if the skip is removed, the issue should be closed?